Makefile

```diff
@@ -128,8 +128,9 @@ container-push: container
 
 .PHONY: container-release
 container-release: container
+	# https://git-scm.com/docs/git-tag#Documentation/git-tag.txt---points-atltobjectgt
 	@docker tag $(DOCKER_REPO):$(VCS_BRANCH)-$(BUILD_DATE)-$(VERSION) $(DOCKER_REPO):$(VERSION)
-	docker push $(DOCKER_REPO):$(VERSION)
+	docker push $(DOCKER_REPO):$(strip $(shell [ -d .git ] && git tag --points-at HEAD))
```

Comment:
The tag getting pushed here `$(DOCKER_REPO):$(strip $(shell [ -d .git ] && git tag --points-at HEAD))` does not match the tag created above `$(DOCKER_REPO):$(VERSION)`